PurchasesCoreSwift/Attribution/AttributionTypeFactory.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AttributionTypeFactory.swift
```

Comment:
Let's split the classes out of this file into their own 
📖 -> 📃📃📃

Reply:
Yep, agree with you